### PR TITLE
add my profile info

### PR DESCRIPTION
**Pull request by AhnafAhamed**
Please add my profile info to the project.
Thanks 🤗

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/184"></a>

### CODE REVIEW

**Review by Flolight — CHANGES_REQUESTED** [review]
Be careful, you forgot to change the Twitter url with yours.

File: public/data/ahnafahamed.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "avatar": "https://avatars.githubusercontent.com/u/42427928?v=4",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/AhnafAhamed",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/eddiejaoude",
```
Comment (FayasNoushad) [inline public/data/ahnafahamed.json:13]: If you have twitter account change you twitter url else remove the dict ( json ).

Comment (AhnafAhamed) [reply public/data/ahnafahamed.json:13]: I'll fix that, thank you

**Review by FayasNoushad — APPROVED** [review]

File: public/data/ahnafahamed.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Ahnaf Ahamed",
+  "bio": "Front-end developer",
+  "avatar": "https://avatars.githubusercontent.com/u/42427928?v=4",
```
Comment (FayasNoushad) [inline public/data/ahnafahamed.json:4]: Change Avatar to `https://github.com/AhnafAhamed.png` for auto updating.

Comment (AhnafAhamed) [reply public/data/ahnafahamed.json:4]: I'll do that, but when I do that I can't see my profile picture. Please enlighten me. Thanks

**Review by FayasNoushad — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**